src/Paket.Core/NupkgWriter.fs

```diff
@@ -141,7 +141,7 @@ let corePropsDoc (core : CompleteCoreInfo) =
     let xsi = XNamespace.Get "http://www.w3.org/2001/XMLSchema-instance"
     let root =
         XElement
-            (ns + "Relationships", XAttribute(XName.Get "xmlns", ns.NamespaceName),
+            (ns + "coreProperties", XAttribute(XName.Get "xmlns", ns.NamespaceName),
```

Comment:
I noticed this when comparing a package created with NuGet.exe vs. one created with Paket.exe, I'm assuming "Relationships" was a copy-paste error from the code for the .rels file.